poc/test_opaque_ake.sage

```diff
@@ -33,6 +33,7 @@ def test_core_protocol_serialization():
 
     config = default_opaque_configuration 
     group = config.group
+    (_, _) = group.key_gen()
```

Comment:
what does that do?

Reply:
It fixes the deterministic test vector issue @stef pointed out. I'll address this (by removing it) in a future change that _does not_ make any protocol changes.